docs/build-nix.md

```diff
@@ -0,0 +1,18 @@
+# Reproducible image building using Nix
+
+`nerdctl build-nix` (EXPERIMENTAL) allows [reproducible image building](https://en.wikipedia.org/wiki/Reproducible_builds) using Nix.
+
+## Example
+
+See [`../examples/build-nix/nginx`](../examples/build-nix/nginx)
+
+```
+nerdctl build-nix
+nerdctl run -d -p 8080:80 nginx:nix
+```
```

Comment:
Maybe the expected digest of the result image should be documented so that users can confirm that `build-nix` is reproducible.